packages/automerge-repo-react-hooks/test/useDocument.test.tsx

```diff
@@ -29,14 +29,21 @@ describe("useDocument", () => {
     const handleSlow = repo.create<ExampleDoc>()
     handleSlow.change(doc => (doc.foo = "slow"))
     const oldDoc = handleSlow.doc.bind(handleSlow)
+    const syncDoc = undefined
     handleSlow.doc = async () => {
       await new Promise(resolve => setTimeout(resolve, SLOW_DOC_LOAD_TIME_MS))
       const result = await oldDoc()
+      const syncDoc = result
       return result
     }
+    handleSlow.docSync = () => {
+      return syncDoc
+    }
```

Comment:
Just for readability, I think this would be clearer as
```suggestion
    const oldDoc = handleSlow.docSync.bind(handleSlow)
    const result = oldDoc()
    handleSlow.doc = async () => {
      await new Promise(resolve => setTimeout(resolve, SLOW_DOC_LOAD_TIME_MS))
      return result
    }
    handleSlow.docSync = () => {
      return result
    }
```
(I _think_ that has the same semantics.)